src/System.DirectoryServices.Protocols/tests/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+        netstandard-Windows_NT;
+        netstandard;
```

Comment:
Actually I think if you remove 'netstandard' here it will become Windows only without need for the attribute